### DISCUSSION TITLE
`api_base` should be http://localhost:8080/v1

### DISCUSSION THREAD

**Discussion by johnd0e** [darrenburns/elia#39] [Ideas]
According to the documentation, `api_base` is specified as the full endpoint URL like so:
```
api_base = "http://localhost:8080/v1/chat/completions"
```
However, the term `api_base` generally implies that it's used as the "common" part of the URL, shared by different endpoints. This is exactly how the name is used in other applications.

So, it would be more appropriate to define `base_url` as http://localhost:8080/v1.

**Comment by darrenburns** [comment]
Thanks - I've fixed the docs now.
